crates/router/src/routes/user.rs

```diff
@@ -336,6 +336,24 @@ pub async fn get_user_role_details(
     .await
 }
 
+pub async fn list_user_roles_details(
+    state: web::Data<AppState>,
+    req: HttpRequest,
+    payload: web::Query<user_api::GetUserRoleDetailsRequest>,
+) -> HttpResponse {
+    let flow = Flow::GetUserRoleDetails;
```

Comment:
New flow?

Reply:
Yes we can have, but this flow was not getting used anywhere